components/portfolio/BondsTable.tsx

```diff
@@ -28,8 +34,22 @@ const columns: TableColumn[] = [
   },
 ];
 
+const lookUpBaseAssetPrice = (
```

Comment:
code in this function is used in other places and possibly more in the future. Can it be moved to `util` somewhere and so it can be reused?